traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -243,3 +343,39 @@ func DeleteTestRegions(t *testing.T) {
 		}
 	}
 }
+
+func DeleteTestRegionsByInvalidId(t *testing.T) {
+	i := 10000
+	delResp, _, err := TOSession.DeleteRegion(&i, nil)
+	if err == nil {
+		t.Errorf("cannot DELETE Regions by Invalid ID: %v - %v", err, delResp)
+	}
+}
+
+func DeleteTestRegionsByInvalidName(t *testing.T) {
+	i := "invalid"
+	delResp, _, err := TOSession.DeleteRegion(nil, &i)
+	if err == nil {
+		t.Errorf("cannot DELETE Regions by Invalid ID: %v - %v", err, delResp)
+	}
+}
+
+func GetRegionByInvalidId(t *testing.T) {
+	regionResp, _, err := TOSession.GetRegionByID(10000, nil)
+	if err != nil {
+		t.Errorf("Error!! Getting Region by Invalid ID %v", err)
+	}
+	if len(regionResp) >= 1 {
+		t.Errorf("Error!! Invalid ID shouldn't have any response %v Error %v", regionResp, err)
+	}
+}
+
+func GetRegionByInvalidName(t *testing.T) {
+	regionResp, _, err := TOSession.GetRegionByName("abcd", nil)
+	if err != nil {
+		t.Errorf("Error!! Getting Region by Invalid Name %v", err)
+	}
+	if len(regionResp) >= 1 {
+		t.Errorf("Error!! Invalid Name shouldn't have any response %v Error %v", regionResp, err)
```

Comment:
You don't need to prefix test errors with "Error!! ", the output's gonna put a big "ERROR" in front of it for you

Reply:
Fixed.